src/common/Environments.ts

```diff
@@ -12,7 +12,7 @@ interface Environment {
 }
 
 const defaultRelayHubConfiguration: RelayHubConfiguration = {
-  gasOverhead: 34936,
+  gasOverhead: 37054,
```

Comment:
the overhead of these tests is by far not neglible: its total of 2000

Reply:
yeah, that is weird, we make like 5 checks, how is it so expensive?